code/modules/mob/living/living.dm

```diff
@@ -637,13 +637,14 @@
 /mob/living/resist_grab(moving_resist)
 	. = 1
 	if(pulledby.grab_state)
-		if(!resting && prob(30/pulledby.grab_state))
+		var/resist_chance = (moving_resist == TRUE) ? 10/pulledby.grab_state : 30/pulledby.grab_state
```

Comment:
This var is only used one single time. Defining a var that only gets used one single time is a pretty bad coding practice in the context of byond, as declaring a var has a pretty hefty performance overhead at runtime.
Move the contents of this var into the below prob() call

Reply:
Done.